src/usr/local/www/diag_states_summary.php

```diff
@@ -129,8 +129,11 @@ function build_port_info($portarr, $proto) {
 }
 
 function print_summary_table($label, $iparr, $sort = TRUE) {
+    foreach ($iparr as $ip => $ipinfo) {
+        $iparr[$ip]['ip'] = $ip;
```

Comment:
Why do this? The IP address is already there as a key. This will slow the page down and consume a lot more memory with large state tables.